source/buildVersion.py

```diff
@@ -0,0 +1,45 @@
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2006-2017 NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+

```

Comment:
I would suggest some module level doc string to indicate that "this module is for non-localised version information. It should not contain UI strings or translatable strings.  The versionInfo module is the correct place for that"